libyang/schema.py

```diff
@@ -1132,7 +1132,24 @@ def extensions(self) -> Iterator[ExtensionCompiled]:
             yield ExtensionCompiled(self.context, extension)
 
     def must_conditions(self) -> Iterator[str]:
-        return iter(())
+        mc = ffi.new("struct lysc_must *")
+        mc = lib.lysc_node_musts(self.cdata)
+        if mc == ffi.NULL:
+            return
+        for cond in ly_array_iter(mc):
+            yield c2str(lib.lyxp_get_expr(cond.cond))
+
+    def must_conditions_details(self) -> Iterator[Tuple[str, Optional[str]]]:
+        mc = ffi.new("struct lysc_must *")
+        mc = lib.lysc_node_musts(self.cdata)
+        if mc == ffi.NULL:
+            return
+        for m in ly_array_iter(mc):
+            if not m:
+                continue
+            cond = c2str(lib.lyxp_get_expr(m.cond))
+            err_msg = c2str(m.emsg) if m.emsg != ffi.NULL else None
+            yield cond, err_msg
```

Comment:
I think that we have the same problem as your commit about patterns here (i.e: the API can not be updated with new fields without breaking). So I think we need a Must class, and then methods to access the fields, instead of yielding condition and error message.

Reply:
Done